src/calc.ts

```diff
@@ -0,0 +1,549 @@
+//selecting various elements
+let btns = document.querySelectorAll(".btn");
+let number = document.querySelectorAll(".number");
+const deg = <HTMLElement>document.querySelector("#degree");
+let functionButton = <HTMLElement>document.querySelector(".function-btn");
+const operator = document.querySelectorAll(".operator");
+const equal = document.querySelector("#equal");
+let trigoButton = <HTMLElement>document.querySelector(".trigonometry-function");
+let second = <HTMLElement>document.querySelector("#secondBtn");
+let calScreen = <HTMLElement>document.querySelector(".calculator-screen");
```

Comment:
types can be given to the variables, i.e. HTMLInputElement or HTMLButtonElement etc.